Lib/fontTools/subset/cff.py

```diff
@@ -352,28 +352,45 @@ def processSubr(self, index, subr):
 				if subr_hints.status == 0:
 					hints.last_hint = index
 				else:
-					hints.last_hint = index - 2 # Leave the subr call in
+					hints.last_hint = index - 2  # Leave the subr call in
 
 		elif subr_hints.status == 0:
 			hints.deletions.append(index)
 
 		hints.status = max(hints.status, subr_hints.status)
 
+class StopHintCountEvent(Exception):
+	pass
+
+
+stop_hintcount_ops = ("op_hstem", "op_vstem", "op_rmoveto", "op_hmoveto",
```

Comment:
i'd make this `stop_hintcount_ops` a class variable, it's only used within `_DesubroutinizingT2Decompiler`

Reply:
Right. Thanks for the review! Will fix all these, including the patch for the GlobalSubrs bug.

Reply:
@anthrotype I see that there is already a test, subset_test.py::test_no_hinting_desubroutinize_CFF, which should encounter the AttributeError for `GlobalSubrs`. I can't reproduce this bug, either with this test, or using the command-line "pyftsubset Lobster.subset.otf  --gids=0-11 --desubroutinize --no-hinting". How did you reproduce this problem?

Reply:
@anthrotype Never mind - I realize that the test needs a font with a non-empty GlobalSubrs. Have test case, will add.